app/scripts/seller-transform.js

```diff
@@ -15,13 +15,61 @@ var sellerTransform = (function(_, relatedEntityTransform, commonTransforms) {
 
             if(data.hits.hits.length > 0) {
                 newData._id = _.get(data.hits.hits[0], '_id');
-                newData.telephone = _.get(data.hits.hits[0]._source, 'telephone.name');
-                newData.emailAddress = _.get(data.hits.hits[0]._source, 'email.name');
-                newData.title = 'Seller (' + (newData.telephone || newData.emailAddress || 'Info N/A') + ')';
+                newData._type = _.get(data.hits.hits[0], '_type');
+                newData.telephone = commonTransforms.getClickableObjectArr(_.get(data.hits.hits[0]._source, 'telephone'), 'phone');
+                newData.emailAddress = commonTransforms.getClickableObjectArr(_.get(data.hits.hits[0]._source, 'email'), 'email');
+                var title = undefined;
+                var numPhoneEmails = 0;
+                if(newData.telephone.length > 0) {
+                    title = newData.telephone[0].title;
+                    numPhoneEmails += newData.telephone.length;
+                }
+                if(newData.emailAddress.length > 0) {
+                    if(title) {
+                       title += ", " + newData.emailAddress[0].title;
+                    }
+                    else {
+                        title = newData.emailAddress[0].title;   
+                    }
+                    numPhoneEmails += newData.emailAddress.length;
+                }
+                if(numPhoneEmails > 2) {
+                    numPhoneEmails = numPhoneEmails - 2;
+                    newData.title = 'Seller ' + title + ' (' + numPhoneEmails + ' more)...';
+                }
+                else {
+                    newData.title = 'Seller (' + (title || 'Info N/A') + ')';
+                }
+                
+                newData.subtitle = '';
             }
 
             return newData;
         },
+        phoneEmails: function(data) {
+            var newData = [];
+
+            if(data.hits.hits.length > 0) {
+                
+                telephone = commonTransforms.getClickableObjectArr(_.get(data.hits.hits[0]._source, 'telephone'), 'phone');
+                emailAddress = commonTransforms.getClickableObjectArr(_.get(data.hits.hits[0]._source, 'email'), 'email');
```

Comment:
The telephone and emailAddress variables should be defined locally (using "var") not globally.